container_workflow_tool/utility.py

```diff
@@ -55,6 +55,13 @@ def _get_packager(config):
     return packager
 
 
+def _get_hostname_url(config):
+    # Try to read which packager to use from the config file
+    # default to "https://src.fedoraproject.org"
+    git_url = getattr(config, "hostname_url", "https://src.fedoraproject.org")
```

Comment:
Default to `None` here, so that cwt can still use the old way of cloning git repos if a config does not have the url set.